src/test/rgw/bucket_notification/requirements.txt

```diff
@@ -0,0 +1,4 @@
+nose >=1.0.0
+boto >=2.6.0
+boto3 >=1.0.0
+configparser >=5.0.0
```

Comment:
are these the only requirements?
what about kafka? what about pika (amqp)?

Reply:
Kafka wouldn't be needed as the download gets done in the task.
And I've downloaded Pika library in bootstrap file

Reply:
by "kafka" i meant the kafka client python lib. you need it either here or in the bootstrap

Reply:
Added.

Reply:
how do we decide what goes into `requirements.txt` and what goes into `bootstrap`?